src/DeviceFirmata.cpp

```diff
@@ -0,0 +1,264 @@
+/*
```

Comment:
All of DeviceFirmata.cpp needs to be moved to DeviceFirmata.h or Luni and Bas64 will become hard dependencies of ConfigurableFirmata whether or not the user chooses to use DeviceFirmata.